src/test/java/task10/JsonParserTest.java

```diff
@@ -0,0 +1,88 @@
+package task10;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+import parser.JsonParser;
+import parser.Parser;
+import shop.Cart;
+import shop.RealItem;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class JsonParserTest {
```

Comment:
I don't see test which check that method throw exception. 
There are examples of such tests: https://www.baeldung.com/junit-assert-exception
JsonParser has a method that can throw exception. It is a good candidate for such test